files/es/web/javascript/reference/global_objects/promise/then/index.md

```diff
@@ -32,13 +32,13 @@ p.then(function(value) {
 
 Retorna un `Promise` el cual es determinado por las funciones input:
 
-* Si `alCumplir` o `enRechazo` arroja un error, o retorna un [`Promise`](/es/docs/Web/JavaScript/Referencia/Objetos_globales/Promise) rechazado, `then` retorna un `Promise` rechazado.
-* Si `alCumplir` o `enRechazo` retorna un `Promise` que resuelve, o retorna cualquier otro valor, `then` retorna un `Promise `resuelto.
+- Si `alCumplir` o `enRechazo` arroja un error, o retorna un [`Promise`](/es/docs/Web/JavaScript/Referencia/Objetos_globales/Promise) rechazado, `then` retorna un `Promise` rechazado.
+- Si `alCumplir` o `enRechazo` retorna un `Promise` que resuelve, o retorna cualquier otro valor, `then` retorna un `Promise`resuelto.
```

Comment:
was changed to separate the lists, if you know other way, feel free

```suggestion
* Si `alCumplir` o `enRechazo` arroja un error, o retorna un [`Promise`](/es/docs/Web/JavaScript/Referencia/Objetos_globales/Promise) rechazado, `then` retorna un `Promise` rechazado.
* Si `alCumplir` o `enRechazo` retorna un `Promise` que resuelve, o retorna cualquier otro valor, `then` retorna un `Promise`resuelto.
```

Reply:
This is the style upstream that is enforced, you can use `<!-- -->` to explicitly split a list